### ISSUE TITLE
[Exception]   Your craft path could not be found.

### ISSUE THREAD

**Issue by Chrisedmo** [rsanchez/craft-cli#15]
Sorry bit of a noob here - where am I supposed to run this from? 'm currently in my /craft folder and I've been trying to run craft `show:config` in various locations and I get the same error. Any help?

**Comment by Chrisedmo** [comment]
Hmm it looks like I missed the bit about having to do `craft init` as I am using a custom file structure. I have created the .craft-cli.php file and added in my structures as I believe them to be, but now I'm getting a different error: 

```
PHP Error[8]: Use of undefined constant CRAFT_FRAMEWORK_PATH - assumed 'CRAFT_FRAMEWORK_PATH'
    in file /Users/Documents/Sites/website/private_html/craft/app/services/PathService.php at line 262
#0 /Users/Documents/Sites/website/private_html/craft/app/etc/console/ConsoleApp.php(265): Craft\ConsoleApp->handleError()
#1 /Users/Documents/Sites/website/private_html/craft/app/services/PathService.php(262): Craft\ConsoleApp->handleError()
#2 /Users/Documents/Sites/website/private_html/craft/app/etc/i18n/PhpMessageSource.php(51): Craft\PathService->getFrameworkPath()
#3 /Users/Documents/Sites/website/private_html/craft/app/framework/base/CModule.php(394): Craft\PhpMessageSource->init()
#4 /Users/Documents/Sites/website/private_html/craft/app/etc/console/ConsoleApp.php(220): Craft\ConsoleApp->getComponent()
#5 /Users/Documents/Sites/website/private_html/craft/app/framework/base/CModule.php(103): Craft\ConsoleApp->getComponent()
#6 /Users/Documents/Sites/website/private_html/craft/app/etc/console/ConsoleApp.php(70): Craft\ConsoleApp->__get()
#7 /Users/Documents/Sites/website/private_html/craft/app/framework/base/CApplication.php(170): Craft\ConsoleApp->init()
#8 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/src/bootstrap-craft2.php(62): Craft\ConsoleApp->__construct()
#9 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/src/Application.php(177): require()
#10 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/src/Application.php(128): CraftCli\Application->bootstrap()
#11 unknown(0): CraftCli\Application->onCommand()
#12 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/vendor/symfony/event-dispatcher/EventDispatcher.php(164): call_user_func()
#13 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/vendor/symfony/event-dispatcher/EventDispatcher.php(53): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch()
#14 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/vendor/symfony/console/Application.php(882): Symfony\Component\EventDispatcher\EventDispatcher->dispatch()
#15 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/vendor/symfony/console/Application.php(195): CraftCli\Application->doRunCommand()
#16 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/vendor/symfony/console/Application.php(126): CraftCli\Application->doRun()
#17 phar:///usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar/src/bootstrap.php(20): CraftCli\Application->run()
#18 /usr/local/Cellar/craft-cli/0.8.0/libexec/craft.phar(8): require()
```

**Comment by rsanchez** [comment]
This should be fixed in 0.8.1. `brew update && brew upgrade craft-cli`
